README.md

```diff
@@ -539,6 +541,18 @@ Ember.js section is shown only in directories that contain a `ember-cli-build.js
 | `SPACESHIP_EMBER_SYMBOL` | `🐹 ` | Character to be shown before Ember.js version |
 | `SPACESHIP_EMBER_COLOR` | `210` | Color of Ember.js section |
 
+### Kubectl context (`kubecontext`)
+
+Shows the active kubectl context.
+
+| Variable | Default | Meaning |
+| :------- | :-----: | ------- |
+| `SPACESHIP_KUBECONTEXT_SHOW` | `true` | Current Kubectl context section |
+| `SPACESHIP_KUBECONTEXT_PREFIX` | `$SPACESHIP_PROMPT_DEFAULT_PREFIX` | Prefix before Kubectl context section |
+| `SPACESHIP_KUBECONTEXT_SUFFIX` | `$SPACESHIP_PROMPT_DEFAULT_SUFFIX` | Suffix after Kubectl context section |
+| `SPACESHIP_KUBECONTEXT_SYMBOL` | `k8s: ` | Character to be shown before Kubectl context |
```

Comment:
I guess that's as close to the k8s logo as we can get with an emoticon :D 
I'll change to it.